Lesson2/src/Homework3.java

```diff
@@ -0,0 +1,19 @@
+public class Homework3 {
+
+    int a = 4;
+    int b = 5;
+
+        public static void main (String[]args){
+
+            Homework3 k = new Homework3();
+            k.maxint();
+
+        }
+
+    public void maxint() {
```

Comment:
this method needs to return max of a or b, so you need to change method signature to 

public int maxint(int a, int b) instead of public void maxint() 

and return a if a > b, and return b otherwise